test/language/expressions/arrow-function/arrow/binding-tests-1.js

```diff
@@ -0,0 +1,11 @@
+// Copyright 2015 Microsoft Corporation. All rights reserved.
+// This code is governed by the license found in the LICENSE file.
+
+/*---
+description: this binding tests
+es6id: 14.2
+---*/
+
+var obj = () => {return this;}
+
+assert.sameValue(obj(), this, "this should be lexical for arrow function.");

```

Comment:
Duplicates

- https://github.com/tc39/test262/blob/master/test/language/expressions/arrow-function/lexical-this.js